training_event/serializers.py

```diff
@@ -117,6 +117,15 @@ def validate(self, data):
                 '活动已被学校管理员审核，不可修改')
         return data
 
+    def validate_program(self, program):
+        '''Forbid creat and update event if program is wrong.'''
+        if self.instance and self.instance.program != program:
+            raise serializers.ValidationError("你无权创建培训活动！")
```

Comment:
This branch is for updating the event and the latter is for creation, so the description text for these validation errors should be switched.